react/features/large-video/components/LargeVideo.native.js

```diff
@@ -111,15 +126,60 @@ class LargeVideo extends Component<Props, State> {
         return (
             <DimensionsDetector
                 onDimensionsChanged = { this._onDimensionsChanged } >
-                <ParticipantView
-                    avatarSize = { avatarSize }
-                    participantId = { this.props._participantId }
-                    style = { styles.largeVideo }
-                    useConnectivityInfoLabel = { useConnectivityInfoLabel }
-                    zOrder = { 0 } />
+                <GestureResponderView
+                    onGesture = { this._onGesture }
+                    style = { styles.largeVideo } >
+                    <ParticipantView
+                        avatarSize = { avatarSize }
+                        participantId = { this.props._participantId }
+                        style = { styles.largeVideo }
+                        useConnectivityInfoLabel = { useConnectivityInfoLabel }
+                        zOrder = { 1 }
+                        zoomEnabled = { true } />
+                </GestureResponderView>
             </DimensionsDetector>
         );
     }
+
+    _onGesture: Object => void
+
+    /**
+     * Translates a gesture into an event that other components can handle,
+     * such as a press event or a transform object.
+     *
+     * @private
+     * @param {Object} gesture - The recently detected gesture.
+     * @returns {void}
+     */
+    _onGesture(gesture) {
```

Comment:
(this only makes sense with my comments about "back and forth below"). What if it would be the transform aware component that is listening to the gestures directly ? will it be able to deal with all of the transform and scale limits ?